sixMans/game.py

```diff
@@ -41,6 +50,13 @@ def __init__(
         for observer in self.observers:
             observer._subject = self
     
+        asyncio.create_task(self._notify())
+        # asyncio.create_task(self.create_game_channels())
+        # asyncio.create_task(self.process_team_selection_method())
```

Comment:
Are these left over comments and meant to be removed?

Reply:
I was planning on moving them in here, but they caused problems with async behavior elsewhere. I can remove them.

Reply:
Removed